src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/definition/datatype/ComplexTypes.java

```diff
@@ -24,6 +24,19 @@ public ApiDataType getApiDataType() {
           (PrimitiveApiDataType) keyType.getApiDataType(),
           (PrimitiveApiDataType) valueType.getApiDataType());
     }
+
+    @Override
+    public String name() {
+      return getApiDataType().getApiName();
+    }
+
+    public String keyType() {
```

Comment:
naming: `keyTypeName()` (since it's String and since there's that `name()`) method)